src/component-library/web-table-of-contents/web-table-of-contents.njk

```diff
@@ -1,4 +1,4 @@
-<web-table-of-contents opened>
+<web-table-of-contents opened i18n="{{ i18n.toc | dump }}">
```

Comment:
We may not need the entire toc file if were just getting the title. I wonder if we shouldn't do this.

Reply:
I think i wanted to be universal here and always just pass all the translation strings the component might need. The i18n.toc object looks like this right now: https://github.com/GoogleChrome/web.dev/blob/main/src/site/_data/i18n/toc.yml so its just one property.
But maybe it is premature optimisation and we should pass only title, and add more i18n props if needed over time.

Reply:
This line doesn't need to be here yet right, and it still dumps the whole TOC?

It just seems out of place with the other usage right now.